evap/evaluation/templatetags/evaluation_filters.py

```diff
@@ -10,6 +10,11 @@ def zip_lists(a, b):
     return zip(a, b)
 
 
+@register.filter(name='reverse_order')
+def reverse_order(value):
+    return 999999-abs(value)
```

Comment:
nice. it's now
```
if course.state in ['new', 'prepared', 'editor_approved', 'approved']:
        return course.days_until_evaluation
    elif course.state == "in_evaluation":
        return 100000 + course.days_left_for_evaluation
    else:
        return 200000 + course.days_left_for_evaluation
```
(higher numbers because in the test data we have dates in 2099, so the number of days can reach up to 30,000)